components/dev-tools/autoconf/SPECS/autoconf.spec

```diff
@@ -24,11 +24,18 @@ Source0:   https://ftp.gnu.org/gnu/autoconf/autoconf-%{version}.tar.gz
 BuildRequires: m4 make
 Requires: m4
 
+BuildRequires: perl
 BuildRequires: perl-macros
 BuildRequires: perl(File::Compare)
 BuildRequires: perl(File::Copy)
 BuildRequires: perl(Data::Dumper)
 BuildRequires: perl(Text::ParseWords)
+%if 0%{?suse_version} || 0%{?sle_version}
+Requires: perl-base
+%else
+Requires: perl-interpreter
```

Comment:
The perl-base or perl-interpreter explicit dependency seems unnecessary. `perl(File::Compare)` pulls in those packages on each platform already.

Reply:
I looked through the dependency chain and it doesn't appear that RHEL's perl-file-compare will install the perl binary. I also looked at the Fedora autoconf specfile and they explicitly add perl-interpreter. The SUSE one isn't needed, but I wanted to keep things parallel.

Reply:
If I do `dnf -y install "perl(File::Compare)"` `perl-interpreter` is pulled in on 9.1.

Reply:
> "you SHOULD NOT assume any other packages are present as RPM dependencies and anything brought into the buildroot by the build system can change over time." 

That's from Red Hat's packaging guidelines. They do say that one can usually drop Requires: if we're running autorequires, but I thought we're only doing manual deps.